### PR TITLE
updated background styles for share bar

### PR DESCRIPTION
**Pull request by bhansa**
updated the bg color to `background: rgba(255, 255, 255, 0.7);`, @rajataaron the bg color which you suggested will have less color contrast, so I think white will be better here. Let me know your thoughts.

**Preview link**: https://bhansa.github.io/inpycon2018/index.html

### CODE REVIEW

**Comment by bhansa** [comment]
No issues @vipulgupta2048, I think `position : fixed;` would be okay since it will only interfere with the venue section and the background text is visible there. Let's see what others have to say about it.

**Comment by ananyo2012** [comment]
I actually the like the color Rajat suggested as it goes with the theme. But it isn't much visible over the venue background. White is making it more visible over the venue background. Is it possible to keep it violet over everything except venue where it should be white ? If its too much work I would go with white as it suits over all background.

Also keep it `position: fixed` . Looks good that way.

**Comment by bhansa** [comment]
I had thought that this in mind :), will see if it takes more effort otherwise we'll go with this.

**Comment by vipulgupta2048** [comment]
Hey, since we are working on the share ribbon. 
@ananyo2012 Should we also edit the status that is being shared in FB and Twitter? Like- Tickets are now LIVE. Get yours from www.in.pycon.org

**Comment by bhansa** [comment]
Nice idea @vipulgupta2048, we can change the text for Twitter and email link since FB only have a link.
I am not sure though how many people come to the website and use the share option, they would probably share it from the source itself, in that case, it would be less worthwhile.

**Comment by ananyo2012** [comment]
This is great work @bhansa ! It really looks good now. Just one more small change can you decrease the opacity of `share-bar--dark` to 0.4 ? I believe that was the opacity Rajat used in his image as well.

**Comment by ananyo2012** [comment]
Regarding the share link we can keep updating the content but I also don't know how many people are using it. If we are keen in making people use it we can update it regularly and I can ask the social media team to give the word out in twitter as well. But then someone has to keep updating it with the lastest updates

**Comment by vipulgupta2048** [comment]
I agree most people won't be using this, but then when they do. We should be ready with  it. What I suggest is we update the content this once, with something better.

**Comment by ananyo2012** [comment]
Ok so Prudhvi from Social Media team was also saying to have some sharing options for tickets. @bhansa Can you take this discussion to social media team and decide what content to put and how they can spread the word about it ?

File: custom/css/style.css [hunk 1]
```diff
@@ -1819,19 +1819,36 @@ ul.faq-answer li a:hover{
 }
 
 /* #region share bar ribbon */
+.share-bar-ribbon--light{
+  background: rgba(255, 255, 255, 0.6);
+}
+.share-bar-ribbon--dark{
+  background: rgba(84, 50, 120, 0.6);
```
Comment (ananyo2012) [inline custom/css/style.css]: Change the opacity to 0.4

**Comment by ananyo2012** [comment]
I am going to merge this in. I will be opening a separate issue for share bar content.

**Pull request merged**